libpod/container.go

```diff
@@ -236,8 +236,12 @@ type ContainerConfig struct {
 	CreatedTime time.Time `json:"createdTime"`
 	// Cgroup parent of the container
 	CgroupParent string `json:"cgroupParent"`
+	// HooksDirPath location of oci hooks config files
+	HooksDirPath string `json:"hooks_dir_path"`
```

Comment:
This needs DB changes to persist across reboot

Reply:
Also, should we set a default in NewContainer()?

Reply:
Ok moved it around. this directory is just for test purposes, Undocumented, so I guess we can store it in the database, but maybe it should just be ignored in DB,